samples/geotag.cpp

```diff
@@ -109,7 +110,9 @@ class Position {
   }
 
   Position() = default;
-  virtual ~Position() = default;
+  ~Position() = default;
```

Comment:
Can you make the whole class `final`? Otherwise, I think a non-virtual destructor is considered bad practice, isn't it?

Reply:
Nothing is being inherited here.